app/MwApi/MwApiExtras.php

```diff
@@ -96,8 +105,13 @@ public static function getBlockInfo($wiki, $target, $key = null)
      * @param  string $target - Username to be searched
      * @return array - information about the block
      */
-    public static function getGlobalBlockInfo($target)
+    public static function getGlobalBlockInfo($target,$appealid)
```

Comment:
```suggestion
    public static function getGlobalBlockInfo($target, $appealid)
```

Also please add `$appealid` to doc block